feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -90,24 +107,32 @@ export class ConfiguratorAttributeProductCardComponent implements OnInit {
 
   getProductPrice(): Configurator.PriceDetails | number {
     return (
-      this.product?.valuePrice ||
-      this.product?.quantity ||
-      this.product?.valuePriceTotal
+      this.productCardOptions.product?.valuePrice ||
+      this.productCardOptions.product?.quantity ||
+      this.productCardOptions.product?.valuePriceTotal
     );
   }
 
   extractPriceFormulaParameters() {
-    if (!this.multiSelect) {
+    if (!this.productCardOptions.multiSelect) {
       return {
-        price: this.product.valuePrice,
-        isLightedUp: this.product.selected,
+        price: this.productCardOptions.product.valuePrice,
+        isLightedUp: this.productCardOptions.product.selected,
       };
     }
     return {
-      quantity: this.product.quantity,
-      price: this.product.valuePrice,
-      priceTotal: this.product.valuePriceTotal,
-      isLightedUp: this.product.selected,
+      quantity: this.productCardOptions.product.quantity,
+      price: this.productCardOptions.product.valuePrice,
+      priceTotal: this.productCardOptions.product.valuePriceTotal,
+      isLightedUp: this.productCardOptions.product.selected,
```

Comment:
Issues with strict mode for the lines 124-127

Reply:
Done!